### ISSUE TITLE
🚀 Enhancement:  Ping/Pong in Realtime

### ISSUE THREAD

**Issue by moshOntong-IT** [appwrite/appwrite#8348]
### 🔖 Enhancement description

We should add ping/pong in Realtime. According to [Websocket Client](https://websocket-client.readthedocs.io/en/latest/examples.html#ping-pong-usage), these can serve as a way to keep a connection active even if data is not being transmitted. Why we need this? I have tested the Realtime in appwrite if you redirect on this [Link](https://discord.com/channels/564160730845151244/1256922603306745956). You can see the reason that this feature is needed.

In the SDKs, I think we must add another parameter called `pingInterval`. This will be the basis on periodic to trigger the ping

### 🎤 Pitch

The use case has been tackled [here](https://discord.com/channels/564160730845151244/1256922603306745956)

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by puntiz** [comment]
This should really get picked up else Realtime is unusably broken in terms of reliability. It expects a ton of management from the app developer to ensure it works. I have never had such issues with Firebase

**Comment by moshOntong-IT** [comment]
> This should really get picked up else Realtime is unusably broken in terms of reliability. It expects a ton of management from the app developer to ensure it works. I have never had such issues with Firebase

yeah I think realtime in appwrite not so good. Does not yet a feature of filtering and all the permissions that you settled in documents and files are useless in realtime. I think the team should focus on improving the Realtime😔
